x-pack/test/api_integration/apis/slos/delete_slo.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { cleanup } from '@kbn/infra-forge';
+import expect from '@kbn/expect';
+import type { CreateSLOInput } from '@kbn/slo-schema';
+import { SO_SLO_TYPE } from '@kbn/observability-plugin/server/saved_objects';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { getFixtureJson } from './helper/get_fixture_json';
+import { loadTestData } from './helper/load_test_data';
+import { SloEsClient } from './helper/es';
+
+export default function ({ getService }: FtrProviderContext) {
+  describe('Delete SLOs', function () {
+    this.tags('skipCloud');
+
+    const supertestAPI = getService('supertest');
+    const kibanaServer = getService('kibanaServer');
+    const esClient = getService('es');
+    const sloEsClient = new SloEsClient(esClient);
+
+    let _createSLOInput: CreateSLOInput;
+    let createSLOInput: CreateSLOInput;
+
+    before(async () => {
+      await kibanaServer.savedObjects.clean({ types: [SO_SLO_TYPE] });
```

Comment:
@dominiqueclarke In my tests I use `await kibanaServer.savedObjects.cleanStandardList();`. Do you think I should better switch to what you have? Any advantage/disadvantage using one versus the other?

Reply:
Actually I have following test https://github.com/elastic/kibana/pull/172786/files#diff-ff4d4447ffb42cda48619d4f3c0e08b8fb0f0a78e9cf96961817ac01bbe04367R299, where at the end of all the created SLOs I verify the total number of SLO entries in the SO definition. If any other test wouldn't do proper cleanup, then this would affect my test (not sure if this is even possible). So I'll better call `cleanStandardList` to be on the safe side, if that makes sense.